app/views/courses/ActivateLicensesModal.coffee

```diff
@@ -1,3 +1,4 @@
+# this file is deprecated
```

Comment:
yeah, we can remove this file now. i keep it since i don't know if this deprecated breaks anything or not. but with China usage until now. i think remove this file is safe